package.json

```diff
@@ -26,6 +26,7 @@
     "express": "4.13.0",
     "express-session": "^1.14.2",
     "heapdump": "^0.3.7",
+    "helmet": "^3.8.1",
```

Comment:
Just a note, you'll need to run `npm shrinkwrap` to update the `npm-shrinkwrap.json` file, and commit those changes too.

NPM uses the shrinkwrap file if it's present, and doesn't look at the package.json file. I suspect in the current state it wouldn't actually install helmet.

Reply:
Best way to test this is to `rm -rf node_modules && npm install`, then try to run the app.

Reply:
Thanks, I missed that!